Packs/GitHub/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.py

```diff
@@ -0,0 +1,426 @@
+    def create_user(self, data):
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/Users'
+        return self.http_request(
+            method='POST',
+            url_suffix=uri,
+            data=data,
+        )
+
+    def update_user(self, user_term, data):
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/Users/{encode_string_results(user_term)}'
+        return self.http_request(
+            method='PUT',
+            url_suffix=uri,
+            data=data,
+        )
+
+    def disable_user(self, data):
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/Users/{encode_string_results(data)}'
+        return self.http_request(
+            method='DELETE',
+            url_suffix=uri
+        )
+
+    def get_user_id_by_mail(self, email):
+        user_id = ""
+        user_term = "\"" + email + "\""
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/' \
+              f'Users?filter={encode_string_results("emails")} eq {encode_string_results(user_term)}'
+
+        res = self.http_request(
+            method='GET',
+            url_suffix=uri
+        )
+        res_json = res.json()
+        item = res_json.get('Resources')
+        if item:
+            user_id = item.get('id')
+
+        return user_id
+
+
+def github_handle_error(e):
+    try:
+        resp = e.res
+        error_code = resp.get('errorCode', '')
+        error_message = resp.get('errorCauses', str(e))
+        if isinstance(error_message, list):
+            error_list = []
+            for error in error_message:
+                if isinstance(error, dict):
+                    for key, value in error.items():
+                        error_list.append(f'{key}: {value}')
+                error_message = str('\n '.join(error_list))
+        return error_code, error_message
+
+    except Exception as e:
+        error_code = e.res.status_code
+        error_message = str(e)
+        return error_code, error_message
+
+
+def test_module(client, args):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client: GitHub client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+
+    """result = client.http_request(method,url_suffix)
+    if res.status_code == 200:
+        return 'ok'
+    else:
+        raise Exception"""
+
+    uri = f'scim/v2/organizations/{encode_string_results(client.org)}/Users/1234'
+    res = client.http_request(method='GET', url_suffix=uri)
+    res_text = res.text
+    if 'documentation_url' in res_text:
+        errortext = "URL or Organization Name " + str(client.org) + " Not Found."
+        raise Exception(str(res.status_code) + " " + str(errortext))
+    else:
+        uri = ""
+        res = client.http_request(method='GET', url_suffix=uri)
+        if res.status_code == 200:
+            return 'ok', None, None
```

Comment:
seems that we return results in test_module but those results are not passed to `return_results` and we should return simple text from test_module.